packages/frontend/app/pods/components/pod/settings/relays/relay-table/row/template.hbs

```diff
@@ -0,0 +1,29 @@
+<tr>
```

Comment:
the changes to relay/relay-table are not directly related to the connection pool work, but I decided to refactor because I was in here updating stuff for how to access connection information